Dockerfile

```diff
@@ -132,8 +133,8 @@ EXPOSE 80 443
 
 LABEL name="manageiq" \
           vendor="ManageIQ" \
-          version="Capablanca" \
-          release="latest" \
+          version="Darga" \
+          release=${REF} \
```

Comment:
@Fryguy I don't know if that needs to match.  Maybe @fbladilo or @bazulay can shed some light on that.  If that is the case, I don't know how to make this work without passing another arg.